apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -228,11 +229,7 @@ export const EventSetupTab = (
   });
 
   const { isChildrenManagedEventType, isManagedEventType, shouldLockIndicator, shouldLockDisableProps } =
-    useLockedFieldsManager(
-      eventType,
-      t("locked_fields_admin_description"),
-      t("locked_fields_member_description")
-    );
+    useLockedFieldsManager(eventType, formMethods, t);
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata